### DISCUSSION TITLE
Warn "a client request body is buffered to a temporary file"

### DISCUSSION THREAD

**Discussion by highfly22** [apache/apisix#5329] [Q&A] [answered]
When a route with gprc proxy is used, stdout keeps printing  the warn message "a client request body is buffered to a temporary file" .

I have changed ngx_tpl.lua to set up client_body_buffer_size to 20M. But it doesn't work.

**Comment by tokers** [comment, marked as answer]
You should modify the directive `client_body_buffer_size`.

**Reply by highfly22** [reply]
I tried. But it doesn't work.

**Reply by tokers** [reply]
How large is your client's body?
